### DISCUSSION TITLE
Reolink E1 Outdoor Firmware version: v3.1.0.1643_22122400

### DISCUSSION THREAD

**Discussion by dmiljk** [AT0myks/reolink-fw-archive#5] [Firmware sharing]
Hi,

I contacted Reolink support for confirmation whether E1 Outdoor can fully work with Blue Iris and at the same time asked for a newer firmware since the client didn't report an update. They sent me the link:

https://support.reolink.com/attachments/token/earovP2n9ndB7vFrzTAbtAQsS/?name=IPC_523SD8.1643_22122400.E1-Outdoor.OV05A10.5MP.WIFI1021.PTZ.REOLINK.pak

This is for Hardware No. IPC_523SD8.

Unfortunately I'm not aware of what has been changed in this update.

**Comment by AT0myks** [comment]
Added, thank you for sharing.

**Comment by Usti2000** [comment]
Link not working :-(

**Comment by dmiljk** [comment]
I was hoping the admin will copy the file localy but I provided the original Reolink link as a proof that it is a genuine file.
I've just uploaded the firmware to the Google Drive so you can download it from there and the admin could update the link.

Here it is:
https://drive.google.com/file/d/1V9FwPG22-tFYjDS6-HFhAaLwMgwueEdG/view?usp=sharing

**Comment by Usti2000** [comment]
Thanks mate ! :-)

**Comment by AT0myks** [comment]
@Usti2000 the link works, what's the error you see?

@dmiljk I'm not sure what you mean by copy the file locally.

**Reply by dmiljk** [reply]
The original Reolink link indeed works now, but it definitelly wasn't working this morning when Usti2000 sent the message. I thought it has expired or something.

When I wrote "copying the file locally", I thought that you were downloading firmwares and uploading to the Github account to avoid the possibility of original link expiring.

Regards

**Reply by AT0myks** [reply]
I see, I guess they had some downtime. As far as I know their links don't expire.

GitHub is not intended for file hosting, that's why it's just the links that are archived here.

**Comment by dmiljk** [comment]
Ah, I see... Thanks for the clarification and the service.
